common.py

```diff
@@ -45,23 +55,72 @@ def nested_process_search(self, criteria, cb_conn, base_query):
 
         elif self.product == "cbth":
             return cbth.nested_process_search(cb_conn, criteria, base_query)
+        
+        elif self.product == "defender": 
+            return defender.nested_process_search(cb_conn, criteria, base_query)
```

Comment:
```suggestion
            return defender.nested_process_search(conn, criteria, base_query)
```

Should this variable be renamed as above?

Reply:
I renamed all the instances in that method from ```cb_conn``` to ```conn```.